Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.yml

```diff
@@ -6,7 +6,7 @@ sectionOrder:
 - Connect
 - Collect
 configuration:
-- display: Host - IP (x.x.x.x)
+- display: Server URL
```

Comment:
it's may confusing as currently the url should be without the `https://`
see relevant code in the `main` function

or change the code according